<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParfumForm</class>
 <widget class="QWidget" name="ParfumForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parfüm Detayları</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ParfumForm {
    background-color: #1e293b;
}
QLabel {
    color: #f8fafc;
    font-size: 14px;
    font-weight: bold;
    padding: 5px; /* Increased padding */
}
QLineEdit, QTextEdit, QComboBox, QDoubleSpinBox, QDateEdit, QSpinBox {
    border: 1px solid #475569;
    border-radius: 6px;
    padding: 8px;
    background-color: #334155;
    color: #f8fafc;
    font-size: 14px;
    min-height: 35px; /* Ensure a good minimum height */
}
QLineEdit:focus, QTextEdit:focus, QComboBox:focus, QDoubleSpinBox:focus, QDateEdit:focus, QSpinBox:focus {
    border: 2px solid #3b82f6;
}
QPushButton {
    background-color: #3b82f6;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 18px;
    font-weight: bold;
    min-width: 120px;
    min-height: 36px;
}
QTextEdit {
    min-height: 80px;
}
QLabel[isTitle="true"] {
    font-size: 18px;
    color: #3b82f6;
    margin-bottom: 10px;
}
QFormLayout {
    spacing: 20px; /* Increased spacing */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QLabel" name="formTitleLabel">
     <property name="text">
      <string>PARFÜM BİLGİLERİ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px; color: #3b82f6; font-weight: bold; margin-bottom: 10px; padding: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="adLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Parfüm Adı:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="adLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="markaLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Marka:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="markaComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="turLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Tür:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="turComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="boyutLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Boyut (ml):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="boyutSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="suffix">
        <string> ml</string>
       </property>
       <property name="maximum">
        <double>1000.00</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="kalanMiktarLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Kalan Miktar (ml):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="kalanMiktarSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="suffix">
        <string> ml</string>
       </property>
       <property name="maximum">
        <double>1000.00</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="tarihLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Edinme Tarihi:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="tarihDateEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="fiyatLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Fiyat:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="fiyatSpinBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="suffix">
        <string> ₺</string>
       </property>
       <property name="maximum">
        <double>100000.00</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="kokuNotlariLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Koku Notaları:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="kokuNotlariTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="sezonLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Sezon Önerisi:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="sezonComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="durumLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Durum Önerisi:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="durumComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="aciklamaLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Açıklama:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QTextEdit" name="aciklamaTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="fotoLabel">
       <property name="styleSheet">
        <string notr="true">padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>Şişe Fotoğrafı:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="fotoLayout">
       <item>
        <widget class="QLineEdit" name="fotoPathLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fotoBrowseButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">min-width: 80px;</string>
         </property>
         <property name="text">
          <string>Gözat</string>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QLabel" name="foto_preview">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px dashed #ccc;
background-color: #f9f9f9;</string>
       </property>
       <property name="text">
        <string>Fotoğraf yok</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <setColumnStretch>0, 1</setColumnStretch>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>İptal</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 